README.md

```diff
@@ -160,6 +160,23 @@ There are a few options for enabling it:
     logging.getLogger('stripe').setLevel(logging.DEBUG)
     ```
 
+### Accessing response code and headers
+
+You can access the HTTP response code and headers using the `last_response` property of the returned resource. 
+
+```python
+customer = stripe.Customer.retrieve(
+    "cus_123456789",
+    api_key="sk_test_...",
+    stripe_account="acct_...",
+    stripe_version="2019-02-19"
```

Comment:
Remove the 3 extra options, they aren't relevant to your example here and are confusing (why the 2019 version) right?